tests/test_boolean.py

```diff
@@ -324,217 +195,419 @@ def test_parse_side_by_side_symbols_with_parens_raise_exception(self):
+        assert algebra.FALSE == algebra.FALSE
+
+        assert algebra.TRUE is not algebra.FALSE
+        assert algebra.FALSE is not algebra.TRUE
+
+        assert algebra.TRUE != algebra.FALSE
+        assert algebra.FALSE != algebra.TRUE
+
+        assert bool(algebra.TRUE) is True
+        assert bool(algebra.FALSE) is False
+
+        assert algebra.TRUE == True
+        assert algebra.FALSE == False
 
     def test_literals(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.literals, set())
-        self.assertEqual(algebra.FALSE.literals, set())
+
+        assert algebra.TRUE.literals == set()
+        assert algebra.FALSE.literals == set()
 
     def test_literalize(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.literalize(), algebra.TRUE)
-        self.assertEqual(algebra.FALSE.literalize(), algebra.FALSE)
+
+        assert algebra.TRUE.literalize() == algebra.TRUE
+        assert algebra.FALSE.literalize() == algebra.FALSE
+
+        assert algebra.TRUE.literalize() != algebra.FALSE
+        assert algebra.FALSE.literalize() != algebra.TRUE
 
     def test_simplify(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.simplify(), algebra.TRUE)
-        self.assertEqual(algebra.FALSE.simplify(), algebra.FALSE)
+
+        assert algebra.TRUE.simplify() == algebra.TRUE
+        assert algebra.FALSE.simplify() == algebra.FALSE
 
     def test_dual(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.dual, algebra.FALSE)
-        self.assertEqual(algebra.FALSE.dual, algebra.TRUE)
+
+        assert algebra.TRUE.dual ==  algebra.FALSE
+        assert algebra.FALSE.dual ==  algebra.TRUE
 
     def test_equality(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE, algebra.TRUE)
-        self.assertEqual(algebra.FALSE, algebra.FALSE)
-        self.assertNotEqual(algebra.TRUE, algebra.FALSE)
+
+        assert algebra.TRUE == algebra.TRUE
+        assert algebra.FALSE == algebra.FALSE
+
+        assert     algebra.TRUE != algebra.FALSE
+        assert not algebra.TRUE == algebra.FALSE
 
     def test_order(self):
         algebra = BooleanAlgebra()
-        self.assertTrue(algebra.FALSE < algebra.TRUE)
-        self.assertTrue(algebra.TRUE > algebra.FALSE)
+
+        assert algebra.FALSE < algebra.TRUE
+        assert algebra.TRUE > algebra.FALSE
 
     def test_printing(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(str(algebra.TRUE), '1')
-        self.assertEqual(str(algebra.FALSE), '0')
-        self.assertEqual(repr(algebra.TRUE), 'TRUE')
-        self.assertEqual(repr(algebra.FALSE), 'FALSE')
 
+        assert str(algebra.TRUE) == '1'
+        assert str(algebra.FALSE) == '0'
 
-class SymbolTestCase(unittest.TestCase):
+        assert repr(algebra.TRUE) == 'TRUE'
+        assert repr(algebra.FALSE) == 'FALSE'
 
-    def test_init(self):
```

Comment:
As far as I could tell, these were not tests: no assert statements of any kind. If they were to fail, that would be ugly tracebacks, not test results. Besides, that is just assignment under the hood, so don't see what is being tested.

On the other hand, the `init` part of the test name is confusing and might suggest that this is part of some test setup, which it is not. So, removed.